go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/opencontainers/distribution-spec
```

Comment:
Should this be based off `go mod init specs-go/specs` As there are no go files in the root or packages..

resulting in `module specs-go/specs`

Reply:
have you tested it over in another repo? I didn't try this out..

Reply:
We can evolve it similar to https://github.com/opencontainers/runtime-spec/issues/1069